english/run_out_the_door/run_out_the_door.md

```diff
@@ -24,3 +24,5 @@ You...
 Decide to [start dancing bachata](../start-dancing/startDancing.md) and everybody start to clap and cheer you up.
 
 [Trip and stumble into a pitch-black cave.](grue/grue.md)
+
+Forget about everything and go and find a good pub feed such as a [Chicken schniztel](../schniztel/schnitzel.md) 
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.